src/tir/ir/data_layout.cc

```diff
@@ -182,15 +183,20 @@ Layout Layout::Split(const LayoutAxis& axis, size_t target_pos, int32_t factor)
 int32_t Layout::FactorOf(const LayoutAxis& axis) const {
   if (!defined()) return -1;
   const LayoutAxis& sub = axis.ToSubordinate();
-  if (!this->defined()) return -1;
+
+  int32_t factor = 1;
+  bool has_sub = false;
   for (const IterVar& itvar : operator->()->axes) {
     if (sub == LayoutAxis::Get(itvar)) {
```

Comment:
Not related to this PR, may I know what's the `sub` cases for an axis?

Reply:
TVM treat axis as `prime` and `sub`, `has_sub` means it's blocking format instead of plain.